### PR TITLE
update Analysis/README.md

### PR DESCRIPTION
**Pull request by jwesterd-redhat**

### CODE REVIEW

**Review by djdanielsson — APPROVED** [review]
fixed a few little things

File: roles/analysis/README.md [hunk 1]
```diff
@@ -1,25 +1,61 @@
...
+|-----------------------|------|-------------------------|-------------------------------------------------|
+| analysis_packages_el7 | List | leapp-upgrade             | RPMS that need to be installed for IPU to RHEL7 | 
+| analysis_repos_el7    | List | rhel-7-server-extras-rpms | Repo to be enabled for IPU to RHEL7             |
+| analysis_packages_el8 | List | leapp-upgrade             | RPMS that need to be installed for IPU to RHEL8 | 
+| analysis_repos_el8    | List | rhel-7-server-extras-rpms | Repo to be enabled for IPU to RHEL7             |
+| leapp_upgrade_type    | String  | disconnnected             | Whether connected to RHSM/CDN or not            |
+| leapp_preupg_opt      | String  | --no-rhsm when leapp_upgrade_type is connected | Upstream repository usage - whether to use RHSM | 
+| leapp_metadata_url      | String | | URL to the leapp metadata, usually over https   |
+| leapp_enable_repos_args | String | | --enablerepo (leapp_repos_enabled) or blank                         |
+| leapp_repos_enabled    | String | satellite-client-6-for-rhel-{{ ansible_distribution_major_version | int + 1 }}-x86_64-rpms | Satellite repo for the satellite client |
```
Comment (swapdisk) [inline roles/analysis/README.md:37]: This is not rendering correctly because of the extra  `|` char.

**Review by swapdisk — CHANGES_REQUESTED** [review]

File: roles/analysis/README.md (hunk 1 above)
Comment (djdanielsson) [reply roles/analysis/README.md:37]: this looks to be rendering correctly from what I can tell

File: roles/common/README.md [hunk 2]
```diff
@@ -1,21 +1,20 @@
...
-| log_file                | "{{ log_directory }}/ripu.log" | Local log filename. When a playbook job finishes, a timestamp suffix is appended to the end of the filename. |
-
-## Example playbook
-
-This is a common role included by the `analysis` and `upgrade` roles. It does not need to be explicitly included in your playbook.
-
-## License
-
-MIT
+# Logging
```
[the remark this answers is not in the document]
Comment (swapdisk) [reply roles/common/README.md]: Fixed with d09de249fc0169638001935561663bc488333cd8.

File: roles/analysis/README.md (hunk 1 above)
Comment (swapdisk) [reply roles/analysis/README.md:37]: Actually, looking deeper, I noticed the leapp_repos_enabled variable of String type with this default value is commented out in the code. As currently implemented, this variable is List type and default value is `[]`.

Comment (jeffmcutter) [reply roles/analysis/README.md:37]: That is correct, it is a list, and the default is an empty list.

**Pull request closed without merging**